commit ccad71dabceb850b58ff38e77fd206e68f62ce4b
Author: AnmolSun <124231245+AnmolSun@users.noreply.github.com>
Date:   Thu May 4 13:26:14 2023 +0530

    TEZ-4492: Update Bowerrc to use bower.herokuapp mirror to avoid Bower Registry CERT_EXPIRE issue (BOWER-2608) (#284) (Anmol Sundaram reviewed by Laszlo Bodor)

diff --git a/tez-ui/src/main/webapp/.bowerrc b/tez-ui/src/main/webapp/.bowerrc
--- a/tez-ui/src/main/webapp/.bowerrc
+++ b/tez-ui/src/main/webapp/.bowerrc
@@ -1,7 +1,10 @@
 {
   "directory": "bower_components",
+  "registry": {
+    "search" : [ "http://nexus:8081/repository/bower-proxy/" ]
+  },
   "analytics": false,
   "resolvers": [
-    "bower-shrinkwrap-resolver-ext"
+    "bower-nexus3-resolver"
   ]
 }
